metricbeat/module/ceph/_meta/docs.asciidoc

```diff
@@ -0,0 +1,9 @@
+== ceph Module
+
+This is the ceph Module.
+
+We used code/reference from:
+https://github.com/influxdata/telegraf/blob/master/plugins/inputs/ceph/ceph.go
```

Comment:
I saw that Telegraf uses[ MIT License](https://github.com/influxdata/telegraf/blob/master/LICENSE). So, I understand that there is no problem to copy parts of the Software. Unfortunately, I don't think that it's possible to use as a library. 
To get a better view what kind copy I did, here it's a example. I change the code to generate events.
[Mine](https://github.com/amandahla/beats/blob/master/metricbeat/module/ceph/osdpoolstats/data.go) (line 40)
[Telegraf](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/ceph/ceph.go) (line 462)